src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { useTodos } from '../../context/TodosContext';
+import { Todo } from '../../Types/Todo';
+
+export const TodoForm: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const { addTodo } = useTodos();
+
+  const handleOnSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (title) {
+      const newTodo: Todo = {
+        id: +new Date(),
```

Comment:
It's not a good idea to use Date for id. Consider using more secured way, e.g. uuid tool
https://www.npmjs.com/package/uuid